pkg/container/docker/images.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docker
+
+import (
+	"github.com/pkg/errors"
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+// ImageID returns the image ID for a given image[:tag]
+func ImageID(image string) (string, error) {
+	cmd := exec.Command("docker", "images", "-q", image)
```

Comment:
we should document that we are relying on this behavior when multiple IDs are returned:
```
$ docker images -q kindest/node
2f48f23e3589
f30a33de05b8
f934a127ad69
4a61fa22c039
4a61fa22c039
2d69a822937a
67409709c324
0d16a995491a
```

```
$ docker images -q kindest/node:latest
2f48f23e3589
```

Reply:
I think that's better to default to `latest` if not tag is specified, at least is what I will expect as an user. Let me change it

Reply:
and seems that's the logic implemented in `docker image inspect` 
```

# docker images weaveworks/weave-npc
REPOSITORY             TAG                 IMAGE ID            CREATED             SIZE
weaveworks/weave-npc   2.5.1               789b7f496034        8 weeks ago         49.6MB
# docker image inspect weaveworks/weave-npc
[]
Error: No such image: weaveworks/weave-npc
```

Reply:
what i mean is, we depend on the ordering of the output of by `docker images`

Reply:
we should at least put a comment at the call site about this

Reply:
this no longer applies